allennlp_models/structured_prediction/dataset_readers/universal_dependencies.py

```diff
@@ -111,7 +111,7 @@ def text_to_instance(
                 [x[0] for x in dependencies], text_field, label_namespace="head_tags"
             )
             fields["head_indices"] = SequenceLabelField(
```

Comment:
```suggestion
            # Semantically, using a ListField[IndexField] here would be more appropriate,
            # but we'll do it like because it's easier.
            fields["head_indices"] = SequenceLabelField(
```